lib/linter-integration.js

```diff
@@ -31,13 +31,15 @@ class Linter {
     }
     this.linter.setMessages(messages.map(match => ({
       type: match.type || 'Error',
-      text: match.message || 'Error from build',
+      text: !match.message && !match.html_message ? 'Error from build' : match.message,
+      html: match.html_message,
```

Comment:
If `message` *and* `html_message` is set, it will set both to the linter [which is an error](https://github.com/steelbrain/linter/wiki/Linter-API#messages).